lib/datasource/maven/util.ts

```diff
@@ -189,6 +189,7 @@ export async function getDependencyInfo(
 
   const pomUrl = getMavenUrl(dependency, repoUrl, path);
   const { xml: pomContent } = await downloadMavenXml(pomUrl);
+  // istanbul ignore if
```

Comment:
Will fix it in separate PR, need to wrap XML parsing in try-catch (which is behavioral change).